blivet/flags.py

```diff
@@ -64,6 +64,10 @@ def __init__(self):
 
         self.multipath_friendly_names = True
 
+        # set to False since blivet doesn't need to account for any
+        # blacklisted devices by default
+        self.blacklisted_devs = False
```

Comment:
I think this shouldn't be a boolean flag, but a configuration item holding the list of regexes like ``INSTALLER_BLACKLIST`` does now.